addons/l10n_it_edi/models/account_edi_proxy_user.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+class AccountEdiProxyClientUser(models.Model):
+    _inherit = 'account_edi_proxy_client.user'
+
+    edi_format_id = fields.Many2one('account.edi.format', required=True)
+    edi_format_code = fields.Char(related='edi_format_id.code', readonly=True)
+
+    _sql_constraints = [
+        ('unique_edi_identification_per_for', 'unique(edi_identification, edi_format_id)', 'This edi identification is already assigned to a user'),
+    ]
+
+    def _prepare_vals_edi_proxy_client(self, response, company, proxy_type, edi_identification, private_pem):
+        vals = super()._prepare_vals_edi_proxy_client(response, company, proxy_type, edi_identification, private_pem)
+        vals['edi_format_id'] = self.env.ref('l10n_it_edi.edi_fatturaPA').id
```

Comment:
I don't understand why you still need that

Reply:
I just don't want to change anything about the l10n_it_edi flow at the moment and I think they still need edi_format_id